### PR TITLE
(docs/apps): Adds documentation for App Service and some permalink changes for other docs

### PR DESCRIPTION
**Pull request by deshmukhmayur**
# Closes 1205

# Explain the feature/fix

## Does this PR introduce a breaking change

Yes. Some of the permalink to the documentation from other SPAs might be broken. I've tried to fix it wherever I could find the old links in home spa and user-group spa.

Need to check if any other SPA requires these changes.

## Screenshots

<details>
<summary>View Screenshots</summary>

| Label | Screenshot |
|---|---|
| Sidebar reordered | Screenshot 2021-07-07 at 6 51 30 PM |
| Deprecation notice on Home Service | Screenshot 2021-07-07 at 6 50 37 PM |
| Apps Service Documentation | Screenshot 2021-07-07 at 6 53 34 PM |

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?

### CODE REVIEW

**Review by hybridx — APPROVED** [review]
Home-SPA changes look good along with other changes

File: packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ We are happy to announce the Beta release for [One Platform](https://beta.one.re
 ## Highlights of this release
 
 - Enhancements to the [Home page](https://beta.one.redhat.com/) for seamless user experience.
-- Enhancements to the [User & Group microservice](https://beta.one.redhat.com/get-started/docs/microservices/user-groups/user-groups-service/) for implementing a consolidated team framework.
+- Enhancements to the [User & Group microservice](https://beta.one.redhat.com/get-started/docs/microservices/user-groups-service/) for implementing a consolidated team framework.
```
Comment (riginoommen) [inline packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md:15]:
```suggestion
- Enhancements to the [User & Group microservice](https://one.redhat.com/get-started/docs/microservices/user-groups-service/) for implementing a consolidated team framework.
```

File: packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ We are happy to announce the Beta release for [One Platform](https://beta.one.re
 ## Highlights of this release
 
 - Enhancements to the [Home page](https://beta.one.redhat.com/) for seamless user experience.
```
Comment (riginoommen) [inline packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md:14]:
```suggestion
- Enhancements to the [Home page](https://one.redhat.com/) for seamless user experience.
```

File: packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ We are happy to announce the Beta release for [One Platform](https://beta.one.re
...
-- Enhancements to the [User & Group microservice](https://beta.one.redhat.com/get-started/docs/microservices/user-groups/user-groups-service/) for implementing a consolidated team framework.
+- Enhancements to the [User & Group microservice](https://beta.one.redhat.com/get-started/docs/microservices/user-groups-service/) for implementing a consolidated team framework.
 - One Portal Migration: Complete [MoD Handovers](https://beta.one.redhat.com/mod-handovers/) SPA Beta UI release.
```
Comment (riginoommen) [inline packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md:16]:
```suggestion
- One Portal Migration: Complete [MoD Handovers](https://one.redhat.com/mod-handovers/) SPA Beta UI release.
```

File: packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ We are happy to announce the Beta release for [One Platform](https://beta.one.re
...
 - Analyst Papers SPA: [Beta UI](https://beta.one.redhat.com/analyst-papers) release that includes Listing, Search, Quote, Quick Preview and Google Drive integration.
```
Comment (riginoommen) [inline packages/documentation-spa/blog/2020-08-18-announcing-one-platform-beta.md:17]:
```suggestion
- Analyst Papers SPA: [Beta UI](https://one.redhat.com/analyst-papers) release that includes Listing, Search, Quote, Quick Preview and Google Drive integration.
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/documentation-spa/docs/microservices/apps-service/app-service.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+---
+id: apps-service
+title: Apps Service
+sidebar_label: Apps Service
+slug: /microservices/apps-service
+---
+***
+
+Apps Servicee is the indexing service which allows you to create and manage individual apps/projects on One Platform. It provides CRUD operations for the app metadata.
```
Comment (riginoommen) [inline packages/documentation-spa/docs/microservices/apps-service/app-service.md:9]:
```suggestion
Apps Service is the indexing service which allows you to create and manage individual apps/projects on One Platform. It provides CRUD operations for the app metadata.
```

Comment (deshmukhmayur) [reply packages/documentation-spa/docs/microservices/apps-service/app-service.md]: Fixed. Thanks for pointing it out.

**Review by riginoommen — APPROVED** [review]

**Pull request merged**